test/compress/template-string.js

```diff
@@ -265,3 +265,39 @@ enforce_single_quotes_and_evaluate: {
         var baz = `Hello ${world()}`;
     }
 }
+
+enforce_single_quote_style: {
+    beautify = {
+        quote_style: 1
+    }
+    input: {
+        var foo = `A single '`;
+        var bar = `A single "`;
+        var baz = `One ' and a "`;
```

Comment:
You need to create additional back quoted strings with `${1+2}` in `enforce_single_quote_style`, `enforce_double_quote_style` and `enforce_any_quote_style` to show that they will always be output in back quotes regardless of `quote_style`.